easybuild/tools/filetools.py

```diff
@@ -122,28 +122,28 @@
 
 EXTRACT_CMDS = {
     # gzipped or gzipped tarball
-    '.gtgz':    "tar xzf %(filepath)s",
+    '.gtgz':    "tar x%(timestamp_tar)szf %(filepath)s",
```

Comment:
Hmm, this makes it a bit hard to read (and it won't work if you use the long option `--touch` instead, see below), so this should become `tar %(timestamp_opt_tar)sxzf %(filepath)s` (and make sure to use `--touch `, i.e. add a space at the end of the `ts_tar` value)

Reply:
see my remark about the long option below